charts/scalar-admin-k8s/README.md

```diff
@@ -0,0 +1,32 @@
+# scalar-admin-k8s
+
+Scalar Admin k8s
+Current chart version is `1.0.0-SNAPSHOT`
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| fullnameOverride | string | `""` | String to fully override scalar-admin-k8s.fullname template |
+| nameOverride | string | `""` | String to partially override scalar-admin-k8s.fullname template (will maintain the release name) |
+| scalarAdminK8s.affinity | object | `{}` | The affinity/anti-affinity feature greatly expands the types of constraints you can express. |
+| scalarAdminK8s.commandArgs | list | `[]` | Arguments of Scalar Admin k8s. You can specify several args as array. |
+| scalarAdminK8s.cronJob.enabled | bool | `false` | Deploy CronJob resource to run Scalar Admin k8s. Disable (using Job resource) by default. |
```

Comment:
Yes. Your understanding is correct.

I agree with you. Indeed, it would be better to control what resources (Job or CronJob) will deploy by using explicit configurations instead of a boolean value. I fixed it in 58af9d688c242fa78cbaed809c636e6b9ed1556c .

I use the value `cronjob` instead of `cron-job` because the resource file name is `cronjob.yaml` in this chart. I think it would be better to keep consistency of terms in this chart.

Thank you for your suggestion!

Reply:
Nice work! 👍